WFP-VAM/prism-app
# Add chart configuration for Cambodia; COUNTRY=cambodia

This PR adds chart configurations for PRISM Cambodia. Unused boundary files were also removed

How to test the feature:
- Run app with country=cambodia
- Select 10-day, 1-month, or 3-month rainfall aggregate or anomaly layer. OR select NDVI, or NDVI anomaly layer
- Click on an admin area
- See the chart
<img width="1495" alt="Screenshot 2023-03-03 at 15 29 32" src="https://user-images.githubusercontent.com/3343536/222671064-10487317-586a-4744-9d3f-b0220f73c2ff.png">
